### PR TITLE
have interactive prompt for indexing fonts

### PR DESCRIPTION
**Pull request by DaveParr**

### CODE REVIEW

**Comment by maelle** [comment]
Shouldn't this happen only when the font table is empty? I'm not sure.

**Comment by maelle** [comment]
I created a conflict, sorry.

Anyhow I think that on top of `interactive` this question should be asked when `nrow(fnt) == 0`?

**Review by maelle — CHANGES_REQUESTED** [review]
see above

**Comment by maelle** [comment]
Yay, thanks! Add yourself as "aut" in DESCRIPTION and then I'll merge.

To do that install the `whoami` package and the `desc` package and run `desc::add_me(role = "aut")`.

**Comment by DaveParr** [comment]
Was just about to start writing a test for this for coveralls stopped being grumpy. Worth doing or shall I just mark it as `#nocov`?

**Comment by maelle** [comment]
ah if you have time it'd be grand! I'd suggest doing it with mocking! Ping me if you have never done that.

**Comment by maelle** [comment]
@DaveParr you included yourself twice! 👯‍♂️

**Comment by maelle** [comment]
@DaveParr do you think you can edit the test (to use mocking) soon? Then I'll merge (no need to test the behavior depending on the user's input)

**Comment by maelle** [comment]
does the build fail locally too?

File: R/zzz.R [hunk 1]
```diff
@@ -14,7 +17,7 @@ is_testing <- function() {
     extrafont::loadfonts("win", quiet = TRUE)
   }
 
-  fnt <- extrafont::fonttable()
```
Comment (DaveParr) [inline R/zzz.R]: @maelle I made a mistake here, this line shouldn't have been changes

Comment (maelle) [reply R/zzz.R]: ok, and once corrected does it pass?

Comment (DaveParr) [reply R/zzz.R]: it builds, it's not tested though, as discussed above :)

**Pull request merged**